src/pages/Topics.tsx

```diff
@@ -0,0 +1,98 @@
+import { useState } from 'react';
+import { useRive, useStateMachineInput } from 'rive-react';
+import { Box, Divider, Heading } from 'theme-ui';
+
+import Seo from '../components/Seo';
+import TopicsCart from '../components/TopicsCard';
+import { isEmpty } from '../utils/isEmpty';
+import { cards } from './../data/cardData';
+
+const STATE_MACHINE_NAME = 'machine';
+const ON_HOVER_INPUT_NAME = 'Hover';
+
+const HomagePage = () => {
+  const { RiveComponent, rive } = useRive({
+    autoplay: true,
+    stateMachines: STATE_MACHINE_NAME,
+    src: '/solar.riv',
+  });
+
+  const onHoverInput = useStateMachineInput(
+    rive,
+    STATE_MACHINE_NAME,
+    ON_HOVER_INPUT_NAME,
+  );
```

Comment:
We have these duplicated codes in HomePage, so it's better to create a separate component.